pipelines/jobs/configurations/jdk21.groovy

```diff
@@ -0,0 +1,56 @@
+targetConfigurations = [
+        'x64Mac'      : [
+                'temurin'
+        ],
+        'x64Linux'    : [
+                'temurin'
+        ],
+        'x64AlpineLinux' : [
+                'temurin'
+        ],
+        'x64Windows'  : [
+                'temurin'
+        ],
+        'x32Windows'  : [
+                'temurin'
+        ],
+        'ppc64Aix'    : [
+                'temurin'
+        ],
+        'ppc64leLinux': [
+                'temurin'
+        ],
+        's390xLinux'  : [
+                'temurin'
+        ],
+        'aarch64Linux': [
+                'hotspot',
```

Comment:
need hotspot?

Reply:
So it's based on jdk-20, and we build that there: https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk20/